Representable/representable
# entry page state views only

users can now only draw a community when going through the flow of selecting a state. closes #361 

**changes**
- going to /entry/ now redirects to the state selection on home page
- clicking on the header "Add Community" button now redirected to state selection on home page
- removed listening for geocoder response on entry page
- geo.js code now uses mapbox more efficiently
- cleaned up some deprecated css

**to test**
- click on "add community" header button
- try to go to "/entry/" manually
- go to entry page, check that it loads correctly when using geocoder
